packages/strapi-plugin-i18n/services/content-types.js

```diff
@@ -101,16 +115,36 @@ const copyNonLocalizedAttributes = (model, entry) => {
  * @returns {string[]}
  */
 const getLocalizedAttributes = model => {
-  return getVisibleAttributes(model).filter(
-    attributeName => isLocalizedAttribute(model, attributeName)
+  return getVisibleAttributes(model).filter(attributeName =>
+    isLocalizedAttribute(model, attributeName)
   );
-}
+};
+
+/**
+ * Fill non localized fields of an entry if there are nil
+ * @param {Object} entry entry to fill
+ * @param {Object} relatedEntry values used to fill
+ * @param {Object} options
+ * @param {Object} options.model corresponding model
+ */
+const fillNonLocalizedAttributes = (entry, relatedEntry, { model }) => {
```

Comment:
if realtedEntry is null. Which it can be in the case of a single type in getAndValidateRelatedEntity this will throw.